sources/packages/backend/apps/api/src/services/application-restriction-bypass/application-restriction-bypass.service.ts

```diff
@@ -1,19 +1,46 @@
 import { Injectable } from "@nestjs/common";
+import { InjectRepository } from "@nestjs/typeorm";
 import {
+  Application,
   ApplicationRestrictionBypass,
   ApplicationStatus,
-  RecordDataModelService,
+  NoteType,
+  OfferingIntensity,
+  RestrictionActionType,
+  StudentRestriction,
+  User,
 } from "@sims/sims-db";
-import { DataSource, Not } from "typeorm";
+import { Brackets, DataSource, Repository, UpdateResult } from "typeorm";
+import { CustomNamedError } from "@sims/utilities";
+import {
+  ACTIVE_BYPASS_FOR_STUDENT_RESTRICTION_ALREADY_EXISTS,
+  APPLICATION_RESTRICTION_BYPASS_NOT_FOUND,
+  APPLICATION_IN_INVALID_STATE_FOR_APPLICATION_RESTRICTION_BYPASS_CREATION,
+  STUDENT_RESTRICTION_IS_NOT_ACTIVE,
+  APPLICATION_RESTRICTION_BYPASS_IS_NOT_ACTIVE,
+  STUDENT_RESTRICTION_NOT_FOUND,
+} from "../../constants";
+import {
+  BypassRestrictionData,
+  RemoveBypassRestrictionData,
+} from "apps/api/src/services/application-restriction-bypass/application-restriction-bypass.models";
```

Comment:
Please adjuste the import. Are these not supposed to be part of some index in the services?

Reply:
Sorry, moved to index. Thanks!